main/FatesHistoryInterfaceMod.F90

```diff
@@ -5204,14 +5204,14 @@ subroutine define_history_vars(this, initialize_variables)
          upfreq=2, ivar=ivar, initialize=initialize_variables,                 &
          index = ih_ar_canopy_si)
 
-    call this%set_history_var(vname='FATES_GPP_UNDERSTORY',                    &
+    call this%set_history_var(vname='FATES_GPP_USTORY',                        &
```

Comment:
Sorry it's taken a while to get to this.  What was it that predicated the change in the name here?  We'll need a ctsm-side update to change the name in the user_nl_clm files for the `Fates` testmod and `fates_sp` usermod.

Reply:
This was initially (changing `_UNDERSTORY` to `_USTORY`) because with the added `FATES_` some of the variable names ended up being too long... I had just missed a few in the initial history var update...